packages/itwin/measure-tools/src/widgets/MeasurementActionToolbar.tsx

```diff
@@ -435,18 +433,15 @@ export class MeasurementActionToolbar {
   }
 
   private static buildToolbar(measurements: Measurement[], actionItemList: MeasurementActionItemDef[]): React.ReactNode {
+    console.log("buildingToolbar");
+    const toolItems: ToolbarActionItem[] = actionItemList.map((itemDef: MeasurementActionItemDef) => {
+      itemDef.measurements = measurements;
+      return ToolbarItemUtilities.createActionItem(itemDef.id, 1, itemDef.iconSpec, itemDef.label, itemDef.execute);
```

Comment:
whats the '1' ?

Reply:
updated to use index * 10 for item priority